shaders/src/light_indirect.fs

```diff
@@ -515,35 +515,41 @@ vec3 evaluateRefraction(
     // when reading from the cubemap, we are not pre-exposed so we apply iblLuminance
     // which is not the case when we'll read from the screen-space buffer
     vec3 Ft = prefilteredRadiance(ray.direction, perceptualRoughness) * frameUniforms.iblLuminance;
+    float at = 1.0;
 #else
-    vec3 Ft;
-
     // compute the point where the ray exits the medium, if needed
     vec4 p = vec4(getClipFromWorldMatrix() * vec4(ray.position, 1.0));
     p.xy = uvToRenderTargetUV(p.xy * (0.5 / p.w) + 0.5);
 
     // distance to camera plane
     const float invLog2sqrt5 = 0.8614;
     float lod = max(0.0, (2.0f * log2(perceptualRoughness) + frameUniforms.refractionLodOffset) * invLog2sqrt5);
-    Ft = textureLod(light_ssr, vec3(p.xy, 0.0), lod).rgb;
+    vec4 Fat = textureLod(light_ssr, vec3(p.xy, 0.0), lod);
+
+    // if it's a transparent background, shift color components towards white
+    vec3 Ft = mix(vec3(1.0), Fat.rgb, Fat.a);
```

Comment:
I'm trying to understand whether this is correct. It seems to be the only reasonable solution but I'm not entirely comfortable with it yet.